torchvision/transforms/functional_tensor.py

```diff
@@ -129,7 +129,7 @@ def adjust_contrast(img: Tensor, contrast_factor: float) -> Tensor:
     return _blend(img, mean, contrast_factor)
 
 
-def adjust_hue(img, hue_factor):
+def adjust_hue(img: Tensor, hue_factor: float) -> Tensor:
```

Comment:
Forgot to add an `isinstance` check for the type of `img` to be `torch.Tensor`. Will do it in the next commit.

Reply:
Yes, please, something like here: https://github.com/pytorch/vision/blob/270e02ecaf6341269f27523b92c1b72d117aacf4/torchvision/transforms/functional_tensor.py#L635